docs/Inputs.md

```diff
@@ -166,6 +166,18 @@ import { BooleanInput } from 'admin-on-rest';
 
 This input does not handle `null` values. You would need the `<NullableBooleanInput />` component if you have to handle non-set booleans.
 
+You can use the `options` attribute if you want to override any of Material UI's `Toggle` which is included to `<BooleanInput>` attributes:
```

Comment:
s/You can use the `options` attribute if you want to override any of Material UI's `Toggle` which is included to `<BooleanInput>` attributes/You can use the `options` props to pass any option supported by the Material UI `Toggle` components/